lib/gridfs-stream/index.js

```diff
@@ -79,21 +80,28 @@ util.inherits(GridFSBucket, Emitter);
  * @param {object} [options.metadata] Optional object to store in the file document's `metadata` field
  * @param {string} [options.contentType] Optional string to store in the file document's `contentType` field
  * @param {array} [options.aliases] Optional array of strings to store in the file document's `aliases` field
- * @param {boolean} [options.disableMD5=false] If true, disables adding an md5 field to file data
+ * @param {boolean} [options.disableMD5=false] (DEPRECATED) If true, disables adding an md5 field to file data
```

Comment:
I think we use `**Deprecated**` in the majority of option deprecation instances in this codebase, is there any particular reason to use the all caps in parens here and in L118?

Reply:
Nope, I'll change it